src/java.desktop/share/native/libharfbuzz/hb-ot-shape-complex-use-machine.hh

```diff
@@ -375,7 +375,7 @@ hb_iter_with_fallback_t<machine_index_t<Iter>,
 typename Iter::item_t>
 {
         machine_index_t (const Iter& it) : it (it) {}
-        machine_index_t (const machine_index_t& o) : it (o.it) {}
+        machine_index_t (const machine_index_t& o) : hb_iter_with_fallback_t<machine_index_t<Iter>, typename Iter::item_t>(o), it (o.it) {}
```

Comment:
This change does not seem to be for this PR...

Reply:
Yes, this is a harfbuzz related build fix, see initial note. I'll remove it.